### PR TITLE
HAR file import for app execution

### PR DESCRIPTION
**Pull request by snopoke**
## Product Description
(This is an addition to and admin only feature)

Allows users to populate the workflow by importing a [HAR file](https://en.wikipedia.org/wiki/HAR_(file_format))

## Technical Summary
Upload and parse a HAR file to generate the list of workflow steps required to perform the same set of actions.

### Safety story
This is an addon to the app execution feature which I have tested locally and in unit tests. 

### Automated test coverage
Existing + new tests

### QA Plan
None

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/app_execution/data_model.py [hunk 1]
```diff
@@ -60,6 +60,13 @@ class CommandStep(Step):
     selected_values: list[str] = None
     """Selected values for multi-select commands"""
 
+    def to_json(self):
+        data = super().to_json()
+        for key in ["value", "id", "selected_values"]:
```
Comment (MartinRiese) [inline corehq/apps/app_execution/data_model.py:65]: I am reading this right? If any of those three keys is set to None with will remove it from the returned Json?

File: corehq/apps/app_execution/views.py [hunk 2]
```diff
@@ -43,10 +45,15 @@ def new_workflow(request):
         "workflow": EXAMPLE_WORKFLOW, "run_every": 1, "form_mode": const.FORM_MODE_HUMAN
     })
     if request.method == "POST":
-        form = AppWorkflowConfigForm(request.POST)
-        if form.is_valid():
-            form.save()
-            return redirect("app_execution:workflow_list")
+        import_har = request.POST.get("import-har")
+        har_file = request.FILES.get("har_file")
```
Comment (MartinRiese) [inline corehq/apps/app_execution/views.py:49]: Nit: we should probably stick to either dashes or under scores.

File: corehq/apps/app_execution/har_parser.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
+import json
+
+from corehq.apps.app_execution import data_model
+from corehq.apps.app_execution.api import ScreenType, get_screen_type_and_data
+from corehq.apps.app_execution.models import AppWorkflowConfig
+
+NON_FORM_ENDPOINTS = {"navigate_menu_start", "navigate_menu", "get_details"}
+FORM_ENDPOINTS = {"answer", "submit-all"}
```
[the remark this answers is not in the document]
Comment (snopoke) [reply corehq/apps/app_execution/har_parser.py:8]: This aren't implemented yet

File: corehq/apps/app_execution/data_model.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/app_execution/data_model.py:65]: That's correct, it's mainly to keep what's shown to the user simple.

**Comment by snopoke** [comment]
@MartinRiese bump

**Review by MartinRiese — APPROVED** [review]

**Pull request merged**